src/realm/cluster.cpp

```diff
@@ -1033,7 +1028,7 @@ void verify_set(ArrayRef& arr, size_t sz)
             elements.set_parent(&arr, n);
             elements.verify();
 
-            // FIXME: Check uniqueness of elements.
+            // TODO: Check uniqueness of elements.
```

Comment:
Does TODO now mean that it's something we need "to do" before we are "done done"?

Reply:
I see TODO as something we might consider later, but is not needed in order to fulfill the requirements.